README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# duke.duke project template
```

Comment:
Was this accidentally refactored? Seems like a lot of references to "Duke" got erroneously replaced with "duke.duke".